docs/integrations/graphql-api.md

```diff
@@ -54,7 +54,7 @@ For more detail on constructing GraphQL queries, see the [Graphene documentation
 The GraphQL API employs the same filtering logic as the UI and REST API. Filters can be specified as key-value pairs within parentheses immediately following the query name. For example, the following will return only sites within the North Carolina region with a status of active:
 
 ```
-{"query": "query {site_list(region:\"north-carolina\", status:\"active\") {name}}"}
+{"query": "query {site_list(filters: {region: "us-nc\", status: "active"}) {name}}"}
```

Comment:
Remove errant backslash, drop encapsulating dictionary, and clean up formatting.
```suggestion
query {
  site_list(filters: {region: "us-nc", status: "active"}) {
    name
  }
}
```